homeworks/pontazsolt/1_introduction/2_vars_datatypes/fictive_character.py

```diff
@@ -0,0 +1,28 @@
+#Inputs#
+full_name = (input("Full name: ")).strip().upper()           #printed#
+
+age = int(input("Age: "))                        #corrected#
+
+python_experience = input("Python experience in years: ")      #printed#
+
+while True:
+    want_to_be_professional = input("Do you wanna be professional at Python?: ").strip().lower()    #Ternarized#
+    if want_to_be_professional in ["yes", "no"]:
+        break
+    else:
+        print("I can agree only 'yes' or 'no' answer")
+
+
+#Corrections#
+age_in_days = age * 365   #printed#
+
+
+#Ternary operator#
+ambitious = True if want_to_be_professional == "yes" else False             
+
+
+#Introduction/Result#
+if ambitious:
```

Comment:
Mi lenne ha a ternary operatorod felül nem egy True-t vagy False-t adna vissza, hanem a két lenti print közötti különbséget? Ha megnézed, ugyanazt printeled ki, 1 apróságot kivéve:
"wants" vs "does not want".
Ha a ternary kifejezés ezt a két string-et adná vissza, és ezt csak simán beágyaznád a printelt szövegbe a megfelelő helyre akkor:
-elég lenne 1 print
-nem kéne még egy if-else (itt két if-et írtál, de a lényeg ugyanaz)